src/swish/http.ss

```diff
@@ -475,14 +480,14 @@
           (match request
             [`(<request> ,query)
              (define header (http:read-header ip (http-header-limit)))
-             (define-values (params pos)
-               (http:get-content-params&pos ip header))
-             (on-exit (delete-tmp-files params)
-               (http:file-handler ip op request header
-                 (append query params)))
-             (when (keep-alive? header)
-               (advance! ip pos)
-               (http:handle-input ip op))]))]
+             (define opos (port-position op))
+             (let-values ([(params ipos) (http:get-content-params&pos ip header)])
```

Comment:
Let's keep it in define and define-values form to avoid the indentation.

Reply:
Have you seen what `define-values` expands to? It negates any performance benefit we may be trying to obtain here via multiple return values.

If we always return `pos` when content length is provided, we may be able to simplify the code and eliminate the use of multiple return values.

Reply:
I pushed a sketch that may help w/ indentation, but I haven't compared the generated code.

Reply:
Thanks, I didn't realize what define-values generated. I'll keep the let-values.